### PR TITLE
fixed issue #23

### PR DESCRIPTION
**Pull request by kavicastelo**

### CODE REVIEW

File: client/src/app/components/about/about.component.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
-import { Component } from '@angular/core';
+import {Component, ViewEncapsulation} from '@angular/core';
```
Comment (kavicastelo) [inline client/src/app/components/about/about.component.ts:1]: ovverride issue fixed

File: client/src/app/components/home/home.component.scss [hunk 2]
```diff
@@ -446,7 +446,7 @@ nav {
           margin: 15px 0;
         }
         h3:hover{
-          color: #e27614;
+          color: #e27614!important;
```
Comment (kavicastelo) [inline client/src/app/components/home/home.component.scss:449]: issue fixed by adding !import tags to scss

**Review by kavicastelo — COMMENTED** [review]
by kavi

File: client/src/app/components/about/about.component.ts (hunk 1 above)
Comment (kavindu0) [reply client/src/app/components/about/about.component.ts:1]: correct way

File: client/src/app/components/home/home.component.scss (hunk 2 above)
Comment (kavindu0) [reply client/src/app/components/home/home.component.scss:449]: done

File: client/src/app/components/home/home.component.ts [hunk 3]
```diff
@@ -1,11 +1,20 @@
...
+} from '@angular/core';
 import VanillaTilt from 'vanilla-tilt';
 import {DomSanitizer, SafeResourceUrl} from "@angular/platform-browser";
 
 @Component({
   selector: 'app-home',
   templateUrl: './home.component.html',
-  styleUrls: ['./home.component.scss']
+  styleUrls: ['./home.component.scss'],
+  encapsulation: ViewEncapsulation.None
```
Comment (kavindu0) [inline client/src/app/components/home/home.component.ts:17]: correct way to mitigate the issue

File: client/src/index.html [hunk 4]
```diff
@@ -16,6 +16,11 @@
   <link rel="preconnect" href="https://fonts.gstatic.com">
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+<!--  front-->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Source+Code+Pro&display=swap" rel="stylesheet">
```
Comment (kavindu0) [inline client/src/index.html:23]: add google front from link tag

File: client/src/styles.scss [hunk 5]
```diff
@@ -1,7 +1,5 @@
 /* You can add global styles to this file, and also import other style files */
-@import url('https://fonts.googleapis.com/css2?family=Prompt&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=Agbalumo&family=Prompt&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=Edu+TAS+Beginner&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Source+Code+Pro&display=swap');
```
Comment (kavindu0) [inline client/src/styles.scss:2]: changed the font family to whole website

**Review by kavindu0 — COMMENTED** [review]
by kavindu01

**Pull request merged**